src/content/docs/en/reference/api-reference.mdx

```diff
@@ -2124,7 +2124,8 @@ console.log(foo + bar) // [!code focus]
 <Code
   code={code}
   lang="js"
-  transformers={[transformerNotationFocus()]} />
+  transformers={[transformerNotationFocus(),[transformerMetaHighlight()]}
```

Comment:
There is a bogus square bracket here.

```suggestion
  transformers={[transformerNotationFocus(), transformerMetaHighlight()]}
```

Reply:
Great, thanks! Oops! We'll fix this when we decide exactly which transformer(s) we should show!